.github/workflows/ci.yml

```diff
@@ -25,6 +21,12 @@ jobs:
           cache: "yarn"
           node-version: "16"
 
+      # Inject all environment variables from ".env.example" in $GITHUB_ENV
```

Comment:
My 2 cents here, we don't need to copy the `.env.example` file, you can just set the `DOTENV_CONFIG_PATH=./.env.example`

Reply:
Hi @Janther, I actually tried that but for some reason didn't work. I think DOTENV_CONFIG_PATH is for local project only and not github actions.

Reply:
I will merge this now, we can fix this later if there's a way to set the `DOTENV_CONFIG_PATH` without copying the `.env.example` file.